src/Commands/ImportMediaCommand.php

```diff
@@ -102,7 +103,7 @@ protected function listFiles(string $disk, string $directory = '', bool $recursi
     /**
      * Search through the record list for one matching the provided path.
      * @param  string $path
-     * @param  Collection $existingMedia
+     * @param Collection<string|int, Media> $existingMedia
```

Comment:
suggest: I'm pretty sure `Collection<Media>` is sufficient. Otherwise, I checked the Laravel source, they use `array-key` as key type, probably best to keep consistent if the key type must be defined

```suggestion
     * @param Collection<array-key, Media> $existingMedia
```